vbisam/tests/jvntest.c

```diff
@@ -110,7 +110,9 @@ main (int iargc, char **ppargv)
 		}
 		iloop1++;
 		cbuffer [170] = 0;
-		for (iloop = 0; iloop < FILE_COUNT; iloop++)
+	}
+                for (iloop = 0; iloop < FILE_COUNT; iloop++)
+		{
```

Comment:
```c
        while (fgets (cbuffer, 1024, pshandle) != NULL)
        {
                if (iloop1 == 0) {
                        printf ("[%-8.8s] [%-14.14s]\n", cbuffer, cbuffer + 42);
                }
                iloop1++;
                cbuffer [170] = 0;
                for (iloop = 0; iloop < FILE_COUNT; iloop++) {
                        if (iswrite (ihandle [iloop], cbuffer))
                        {
                                printf ("Error %d writing row %d to file %d\n", iserrno, iloop1, iloop);
                                exit (1);
                        }
                }
                if (iloop1 > atoi (ppargv [1])) {
                        break;
                }
        }
```

Reply:
Completed and the pull request sent again too